plugins/com.contrastsecurity.ide.eclipse.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,130 @@
+#Properties file for com.contrastsecurity.ide.eclipse.ui
```

Comment:
File corresponding to plugins/com.contrastsecurity.ide.eclipse.ui/OSGI-INF/l10n/bundle_ja.properties